frontend/src/Components/IndustrySelection.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useEffect, useContext } from "react";
+import { Button, Chip, Box } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { Link } from "react-router-dom";
+import Nav from "./Nav";
+import "../CSS/SecondProgressBarForm.css";
+import { LinearDeterminate } from "./ProgressBar";
+import { AuthContext } from "../Context/AuthContext";
+import axios from "axios";
+import jwt_decode from "jwt-decode";
+import { api } from "../api/api";
+
+const useStyles = makeStyles({
+  root: {
+    display: "flex",
+    justifyContent: "center",
+    flexWrap: "wrap",
+    width: "30%",
+  },
+  button: {
+    height: "60px",
+    width: "245px",
+    borderRadius: "10px",
+    textTransform: "none",
+    fontSize: "20px",
+  },
+});
+
+const IndustrySelection = ({ question, matchedWith }) => {
+  const classes = useStyles();
+  const [industry, setIndustry] = useState([]);
+  const { token } = useContext(AuthContext);
+  const decoded = jwt_decode(token);
+  const [userId, setUserId] = useState(decoded.userId);
+  const [isAnyIndustriesSelected, setIsAnyIndustriesSelected] = useState(false);
+  const [selectedIndustries, setSelectedIndustries] = useState([]);
+
+
+    const getUserProfile = async () => {
+    try {
+      let config = {
+        method: "get",
+        maxBodyLength: Infinity,
+        url: `${api.users.userProfile}/${userId}`,
+        headers: {
+          Authorization: `Bearer ${token}`,
+          "Content-Type": "application/json",
+        },
+      };
+      const response = await axios.request(config);
+
+      const { industry } = response.data;
+      setIndustry(industry);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getUserProfile();
+  }, []);
+
+  const handleSelectIndustry = (industry) => {
+    if (selectedIndustries.length < 5 && !selectedIndustries.includes(industry)) {
+      setSelectedIndustries([...selectedIndustries, industry]);
+    }
+  };
+
+  const handleDeleteIndustry = (industry) => {
+    setSelectedIndustries(selectedIndustries.filter((i) => i !== industry));
+  };
+
+  const handleSelectAnyIndustries = () => {
+    setSelectedIndustries([]);
+    setIsAnyIndustriesSelected(true);
+  };
+
+  const handleDeselectAnyIndustries = () => {
+    setIsAnyIndustriesSelected(false);
+  };
+
+  const handleContinueClick = async () => {
+    try {
+      let data = JSON.stringify({
+        industry,
+      });
+      let config = {
+        method: "patch",
+        maxBodyLength: Infinity,
+        url: `${api.users.userProfile}/${userId}`,
+        headers: {
+          Authorization: `Bearer ${token}`,
+          "Content-Type": "application/json",
+        },
+        data: data,
+      };
+      const response = await axios.request(config);
+      console.log(response.data);
```

Comment:
TODO: Need to add a navigate here for the upcoming flow-3
navigate('mentor/mentee-flow-3')